src/test/java/seedu/recipe/storage/jsonadapters/JsonAdaptedTagTest.java

```diff
@@ -0,0 +1,44 @@
+package seedu.recipe.storage.jsonadapters;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static seedu.recipe.testutil.Assert.assertThrows;
+
+import org.junit.jupiter.api.Test;
+
+import seedu.recipe.commons.exceptions.IllegalValueException;
+import seedu.recipe.model.tag.Tag;
+
+public class JsonAdaptedTagTest {
+    private static final String VALID_TAG_NAME = "healthy";
+    private static final String INVALID_TAG_NAME = "#fatty";
+
```

Comment:
Are there more invalid tags? i.e. whitespace, special characters, leading trailing whitespace, empty

Reply:
This should be done in model tests.